static/css/daccs-css.css

```diff
@@ -168,13 +168,97 @@
     height: 25%;
     font-family: "Helvetica", sans-serif;
     color: #FFFFFF;
-    background-color: #000000;
+    background-color: rgba(0.0, 0.0, 0.0, 0.5);
+}
+
+#home-video{
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+    z-index: -100;
 }
 
 .banner-spacer{
     height: 20%;
 }
 
+.banner-video {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+    z-index: -100;
+}
+
+.banner-video-overlay{
```

Comment:
unused?

Reply:
May keep this for a static image or overlay in place of video for smaller screen sizes.

Reply:
Class is kept in for video.